Chapter4/proRing.erl

```diff
@@ -0,0 +1,18 @@
+-module(proRing).
+-export([ring/2]).
+
+%%Exercise 4-2: The Process Ring
+%Not sure what was meant by 'number of messages'
+ring(F_Pid, Length) ->
```

Comment:
Try to use function clause instead of `if` statements, this will improve readability. 
```erlang
ring(F_Pid, 1 = _Length) ->
    % ...
ring(F_Pid, 0 = _Length) ->
    % ...
ring(F_Pid, Length) ->
    % ....
```
Common receive loop can be then moved to a separate function (ex: `ring_loop/n`)